editor/editor_dock_manager.h

```diff
@@ -126,9 +134,13 @@ class EditorDockManager : public Object {
 	void _move_dock_tab_index(Control *p_dock, int p_tab_index, bool p_set_current);
 	void _move_dock(Control *p_dock, Control *p_target, int p_tab_index = -1, bool p_set_current = true);
 
+	void _update_tab_style(Control *p_dock);
+
 public:
 	static EditorDockManager *get_singleton() { return singleton; }
 
+	void update_tab_style();
```

Comment:
```suggestion
	void update_tab_styles();
```
It updates all tabs.